label/reports/one_label.xml

```diff
@@ -0,0 +1,37 @@
+<odoo>
+    <template id="one_label">
+        <!--        <link rel="stylesheet" href="/label/static/src/css/label.css"/>-->
+        <!-- In order to keep the layout whatever the datas are, everything is inside a div that hide the overflow and occupy all the place -->
+        <div class="o_label_table_cell">
+            <table class="o_label_cell_table">
+                <tr>
+                    <td style="vertical-align: middle; height: 100%;">
+                        <div class="o_label_div_cell">
+                            <table class="o_label_text_table">
+                                <t t-foreach="11" t-as="val" style="margin-bottom:20px;">
```

Comment:
Do we really want margins and paddings defined in pixels ? I fear that it can change the result depending on other factors that could change from one server to another for instance.

Reply:
Normally pixels margin/padding are always working the same if the base layout doesn't change. Which means that if we don't modify the paper format the outcome should be the same in my opinion.